spec/migrations/update_progress_calculation_spec.rb

```diff
@@ -686,4 +690,46 @@ def expect_migrates(from:, to:)
       end
     end
   end
+
+  describe "error during job execution" do
```

Comment:
To match the flow of reading, I would put things in a more natural order:
1. `let_work_packages` first
2. then the before setting behavior
3. then set behavior of "wp working" first as it is the first created in the `let_work_packages` table
4. then set behavior of "wp breaking" second as it is the second created.

It would feel easier and more natural to me.

Reply:
BTW, the other one in `spec/workers/work_packages/apply_statuses_p_complete_job_spec.rb` is already ok.